### PR TITLE
CHANGELOG for optionally ignoring registry port in <server><id> in settings.xml

### PR DESCRIPTION
**Pull request by chanseokoh**
For #2212.

### CODE REVIEW

File: jib-maven-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [unreleased]
 
 ### Added
+- When giving registry credentials in `settings.xml`, specifying port in `<server><id>` is now optional. ([#2135](https://github.com/GoogleContainerTools/jib/issues/2135))
```
Comment (TadCordle) [inline jib-maven-plugin/CHANGELOG.md]: This sounds like something that was required and is now optional? Maybe it should be something like `Support for optional port in <server><id> in settings.xml.`

Comment (chanseokoh) [reply jib-maven-plugin/CHANGELOG.md]: Actually, specifying port was required if an image reference had it. For example, if `<to><image>my-server:1234/awe/some`, it was required to have `<server><id>my-server:1234</id>`.  Now you can say just `<server><id>my-server</id>`. (`<id>my-server:1234` will continue to work.)

Comment (chanseokoh) [reply jib-maven-plugin/CHANGELOG.md]: Perhaps I should say `... is no longer required`?

Comment (TadCordle) [reply jib-maven-plugin/CHANGELOG.md]: In that case maybe this should go under "Changed" instead of "Added".

**Pull request merged**